docs/elements.md

```diff
@@ -428,7 +428,7 @@ def generate(data):
 | ---------------- | ------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `answers-name`   | string              | —        | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question. |
 | `weight`         | integer             | 1        | Weight to use when computing a weighted average score over elements.                                                                                                 |
-| `correct-answer` | float               | special  | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`. If `base` is provided, then this answer must be given in the provided base.         |
+| `correct-answer` | string              | special  | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`. If `base` is provided, then this answer must be given in the provided base.         |
```

Comment:
Is this actually `int` or `string`?

Reply:
string due to different base (according to @eliotwrobson )

Reply:
This will take an integer that's not in base 10 (i.e. something in hex), and to do that it reads it as a string on the backend. Maybe peeling back the curtain too much but the instructor doesn't really have to worry too much about it.

Reply:
Oh right, I forgot about the base (yet again).

Reply:
> Oh right, I forgot about the base (yet again).

Yeah, the backwards compatibility with this element is such a pain.